Mergin/plugin.py

```diff
@@ -913,7 +901,7 @@ def update_client_and_manager(self, mc=None, manager=None, err=None):
     def updateName(self):
         name = self.base_name
         try:
-            if self.mc.server_type() != ServerType.OLD and self.plugin.current_workspace.get("name", None):
+            if self.plugin.current_workspace.get("name", None):
                 name = f"{self.base_name} [{self.plugin.current_workspace['name']}]"
         except AttributeError:
             # self.mc might not be set yet
```

Comment:
I believe `AttributeError` can no longer be raised. How about:
```suggestion
            name = f"{self.base_name} [{self.plugin.current_workspace['name']}]"
        except KeyError:
            # no current workspace
```
?